ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -27,25 +28,22 @@
         <include pattern="/etc/acs-commons/lists/report-parameter-components/jcr:content/list/default_\d+"/>
         <include pattern="/etc/acs-commons/notifications"/>
         <include pattern="/etc/acs-commons/notifications/jcr:content"/>
+        <include pattern="/etc/acs-commons/notifications/rep:policy(/.*)?"/>
         <include pattern="/etc/acs-commons/packagers"/>
         <include pattern="/etc/acs-commons/packagers/jcr:content"/>
         <include pattern="/etc/acs-commons/redirect-maps"/>
         <include pattern="/etc/acs-commons/redirect-maps/jcr:content"/>
+        <include pattern="/etc/acs-commons/redirect-maps/rep:policy(/.*)?" />
         <include pattern="/etc/acs-commons/exporters"/>
         <include pattern="/etc/acs-commons/exporters/jcr:content"/>
         <include pattern="/etc/acs-commons/qr-code"/>
-        <include pattern="/etc/acs-commons/qr-code/jcr:content"/>
-        <include
-            pattern="/etc/acs-commons/qr-code/jcr:content/clientlib-author"
-            mode="merge"/>
-        <exclude
-            pattern="/etc/acs-commons/qr-code/jcr:content/config"
-            mode="merge"/>
+        <include pattern="/etc/acs-commons/qr-code/jcr:content(/.*)?"/>
+        <!-- the user specific configuration should not be touched -->
+        <exclude pattern="/etc/acs-commons/qr-code/jcr:content/config(/.*)?"/>
         <include pattern="/etc/acs-commons/instant-package"/>
-        <include pattern="/etc/acs-commons/instant-package/jcr:content"/>
-        <include pattern="/etc/acs-commons/instant-package/jcr:content/instant-package-image.png"/>
-        <include pattern="/etc/acs-commons/instant-package/jcr:content/config"/>
-
+        <include pattern="/etc/acs-commons/instant-package/jcr:content(/.*)?"/>
+        <!-- the user specific configuration should not be touched -->
+        <exclude pattern="/etc/acs-commons/instant-package/jcr:content/config(/.*)?"/>
```

Comment:
I placed a node under both `/etc/acs-commons/instant-package/jcr:content/config` and `/etc/acs-commons/qr-code/jcr:content/config` and on deploy the nodes were blown away.  Am I doing something wrong?

Reply:
Indeed, that is very weird, seems like a bug in FileVault, let me investigate.

Reply:
Resolved https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/2045/commits/bc141861b764b2dff80ff9c35c51523e32a0973d